docs/installation/presto-admin-configuration.rst

```diff
@@ -27,7 +41,15 @@ All of the properties are optional. If you include a blank ``config.json`` file,
  "workers": ["localhost"]
  }
 
-The above configuration is for a single-node installation of Presto on the same node that ``presto-admin`` is installed on. You can also specify some but not all of the properties. For example, for a 6 node cluster with default username and port, a sample ``config.json`` would be:
+Note that ``java8_home`` is not set by default.  It only needs to be set if
+Java 8 is in a non-standard location on the Presto nodes.  The property is used
+to tell the presto rpm where to find Java 8.
+
+
```

Comment:
Yes.  If you have only one new line it basically gets ignored and interpreted as a space.  If you actually want two paragraphs, you need two newlines.